### ISSUE TITLE
Allow empty/null station uuid

### ISSUE THREAD

**Issue by eddowh** [gams/openkongqi#12]
All sources require a `uuid` parameter, which is used to identify the dictionary containing the station names for a city.

However, some sources simply cover the city as a whole, without any stations as another layer of specificity.

For example, quite some sources report only city-wide data like Beijing, Shanghai, etc. They do not have a unique station uuid map that identifies any station so technically their `uuid` should either be an empty string or null.

Furthermore, if we come across two sources that each have a station uuid map for a city, our current naming system for uuid isn't exactly going to work. For example, if we have two sources both reporting Beijing's data but they have different station uuids, then the generic uuid `cn/beijing` isn't going to work.

So in conclusion, there are two things to fix:
- allow empty input for `uuid` and modify `openkongqi.stations.get_station_map` to not output on error if key is not found
- make uuid naming system better

**Comment by hrbonz** [comment]
This is not necessary, we can always consider that a city wide number is actually coming from 1 station and name that station. Naming is up to us.
